pinot-controller/src/test/java/org/apache/pinot/controller/helix/ControllerTest.java

```diff
@@ -119,8 +119,8 @@ public class ControllerTest {
 
   protected static HttpClient _httpClient = null;
 
-  protected int _controllerPort;
-  protected String _controllerBaseApiUrl;
```

Comment:
Any specific reason to hide them? IMO it is okay to directly access them

Reply:
This is more to simplify the usage.
Eventually all the usage of `_controllerBaseApiUrl` should go to `getControllerRequestClient()` or `getControllerRequestURLBuilder`, as well as `_brokerBaseApiUrl`.